nomos-core/src/da/blob/info.rs

```diff
@@ -0,0 +1,8 @@
+use super::metadata::Metadata;
+
+pub trait DispersedBlobInfo: Metadata {
+    type BlobId;
+
+    fn blob_id(&self) -> Self::BlobId;
+    fn size(&self) -> usize;
```

Comment:
I'm wondering now if we could have just go for the blob struct directly. But we will see it when composing the final implementation.

Reply:
If we use blob directly, the metadata will need to be optional, I think similar case was with the Certificate.